Romans/Roman Numerals Encoder/index.js

```diff
@@ -0,0 +1,28 @@
+function solution(number) {
```

Comment:
I would suggest you to make the name of the function speak for itself or having a clear name which aligns with what the function is doing